src/main/java/frc/robot/subsystems/Crane.java

```diff
@@ -338,7 +338,9 @@ private Translation2d getDesiredTranslation() {
   }
 
   private Translation2d getDeviation(Translation2d position) {
```

Comment:
This changes `getDeviation()` to `getAbsDeviation()`, but better would be to keep this as-is, and change `scaleAHConstraints()` to use absolute deviation.